### DISCUSSION TITLE
Plotman counts two process for the same plotting job as one.

### DISCUSSION THREAD

**Discussion by simurg79** [ericaltendorf/plotman#429] [Q&A]
I see two processes for each plotting job in plotman:

plot id    k              tmp            dst    wall   phase    tmp     pid   stat      mem    user    sys     io
19a41d66   32   /chia/scratch4   /chia/output    0:17     1:2    54G   25114    SLP     4.7M      0s     0s     0s
19a41d66   32   /chia/scratch4   /chia/output    0:17     1:2    54G   25115    SLP     4.1G    0:22   0:01     2s
5544360e   32   /chia/scratch3   /chia/output    0:37     1:2    80G   24414    SLP     4.7M      0s     0s     0s
5544360e   32   /chia/scratch3   /chia/output    0:37     1:2    80G   24415    SLP     4.1G    0:49   0:02     0s

This breaks a lot of functionality as I cannot use id to access the job. This is happening in debian. Is there a fix for this issue?

**Comment by samtruman** [comment]
> 
> 
> I see two processes for each plotting job in plotman:
> 
> plot id k tmp dst wall phase tmp pid stat mem user sys io
> 19a41d66 32 /chia/scratch4 /chia/output 0:17 1:2 54G 25114 SLP 4.7M 0s 0s 0s
> 19a41d66 32 /chia/scratch4 /chia/output 0:17 1:2 54G 25115 SLP 4.1G 0:22 0:01 2s
> 5544360e 32 /chia/scratch3 /chia/output 0:37 1:2 80G 24414 SLP 4.7M 0s 0s 0s
> 5544360e 32 /chia/scratch3 /chia/output 0:37 1:2 80G 24415 SLP 4.1G 0:49 0:02 0s
> 
> This breaks a lot of functionality as I cannot use id to access the job. This is happening in debian. Is there a fix for this issue?

Same problem with Ubuntu

**Comment by altendky** [comment]
https://github.com/ericaltendorf/plotman/pull/418
